lib/settings/widget_settings/digital_date_widget_settings.dart

```diff
@@ -0,0 +1,109 @@
+
+class DigitalDateWidgetSettingsView extends StatelessWidget {
+  const DigitalDateWidgetSettingsView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final settings = context.read<WidgetStore>().digitalDateSettings;
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Font',
+          builder: (context) {
+            return CustomDropdown<String>(
+              isExpanded: true,
+              value: settings.fontFamily,
+              items: FontFamilies.fonts,
+              onSelected: (family) =>
+                  settings.update(() => settings.fontFamily = family),
+            );
+          },
+        ),
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Font size',
+          builder: (context) {
+            return CustomSlider(
+              min: 10,
+              max: 400,
+              valueLabel: '${settings.fontSize.floor().toString()} px',
+              value: settings.fontSize,
+              onChanged: (value) => settings.update(
+                () => settings.fontSize = value.floorToDouble(),
+              ),
+            );
+          },
+        ),
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Position',
+          builder: (context) {
+            return AlignmentControl(
+              alignment: settings.alignment,
+              onChanged: (alignment) =>
+                  settings.update(() => settings.alignment = alignment),
+            );
+          },
+        ),
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Border',
+          builder: (context) {
+            return CustomDropdown<BorderType>(
+              isExpanded: true,
+              value: settings.borderType,
+              items: BorderType.values,
+              itemBuilder: (context, type) => Text(type.name.capitalize()),
+              onSelected: (value) =>
+                  settings.update(() => settings.borderType = value),
+            );
+          },
+        ),
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Separator',
+          builder: (context) {
+            return CustomDropdown<DateSeparator>(
+              isExpanded: true,
+              value: settings.separator,
+              items: DateSeparator.values,
+              itemBuilder: (context, type) => Text(type.name.capitalize()),
+              onSelected: (value) =>
+                  settings.update(() => settings.separator = value),
+            );
+          },
+        ),
+        const SizedBox(height: 16),
+        LabeledObserver(
+          label: 'Date Format',
+          builder: (context) {
+            return CustomDropdown<DateFormat>(
+              isExpanded: true,
+              value: settings.format,
```

Comment:
This shows `dayMonthYear` in the dropdown which not good because people are not used to this. They are used to `dd/mm/yyyy` format. We have separator info we can render these with that.

Reply:
Actually I have written this as `dayMonthYear` because if we write `dd/mm/yyyy` like this because if we use this `dd/mm/yyyy` then it already contains separator and I think that it limits when user have to change options.

Reply:
We can use the separator selected by the user instead of /. It doesn't matter that much since its a template of how it would look like! But showing dayMonthYear might be more confusing.